google/detectors/credentials/ncrack/src/test/java/com/google/tsunami/plugins/detectors/credentials/ncrack/NcrackWeakCredentialDetectorTest.java

```diff
@@ -158,6 +158,7 @@ public void run_whenATestersReportsValidCredential_returnsFinding() {
                         .setDescription(
                             "Well known or weak credentials are detected for 'http' service on"
                                 + " port '80'.")
+                        .setCvssV3("7.5")
```

Comment:
This is breaking unit tests. You could either move this change to your new merge request https://github.com/google/tsunami-security-scanner-plugins/pull/227 or combine that one with this one to make the tests pass.

Reply:
The mistake has been fixed in the new commit. Keeping #227 separate because it is a different feature.